### DISCUSSION TITLE
Can SnipeIT generate Assets, based on stock Labels?

### DISCUSSION THREAD

**Discussion by gdsgsg** [snipe/snipe-it#12791] [General]
### Is your feature request related to a problem? Please describe.

There is no problem. We want to implement SnipeIT to help us identify assets based on the Labels that come from the factory.
Since no official documentation is sharing this, i was wondering if someone from the community may share if such things were done in the past. 

### Describe the solution you'd like

Ideally, we would like to enable Snipe it, to create assets based on Label information. And if scanned a label, to be able to share what type of device this may be, and its respective available information.

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
